app/models/question.rb

```diff
@@ -0,0 +1,10 @@
+class Question < ActiveRecord::Base
+#has_many :indicators through: :indicator_questions
+
+  validates_presence_of :question
+
+  # QUESTION: Should we allow some way to order the questions?:
```

Comment:
So we should've asked Kim about whether or not order of questions matters lol
So I looked at what they did last year and they just used the id (primary key) as the question number. I think that way is pretty bad, especially when you want to change the order of the questions.
I think we will need another field called question_number that denotes the order.